css/slider/slider.less

```diff
@@ -159,6 +159,8 @@
 			display: block;
 
 			height: 1em;
+			margin: -15px;
```

Comment:
Yeah, I'm generally not a fan of using negative margin values. Rather change the position of the containing `div` below. I.e. in the `.sow-slide-nav-next` selector set `right: 5px;` and for `.sow-slide-nav-prev` set `left: 5px;`

Reply:
Super, thanks for pushing for a better solution.